its/plugin/projects/externalreport/rubocop/yaml-issue.rb

```diff
@@ -0,0 +1,3 @@
+def security
+  YAML.load('--- foo')
+end
```

Comment:
move this file in a sub-folder, so the paths will contain a path separator and will be tested on windows